monitoring/mongodb/alerts.yaml

```
@@ -139,3 +139,24 @@ groups:
     annotations:
       description: 'MongoDb has low disk space'
       summary: 'MongoDb has low disk space'
+  
+  - alert: MongoDbNodeRecovering
+    expr: |
+      sum by (pod) (mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state="3"} >= 1)
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      description: "Mongodb node in `{{ $labels.pod }}` is in the 'recovering' state for more than 10 minutes."
```

I would suggest having first a warning after 1h, and then the critical alert (after 24h?).
I also think our alert descriptions should be helpful for the reader in general (someone without deep knowledge about the product/mongodb), but please challenge that if you disagree.

```suggestion
      description: "The Mongodb instance \"`{{ $labels.pod }}`\" is in the 'RECOVERING' state for over an hour. The instance may not be able to join the replica set if the platform ingests a large number of operations during this time. This alert is expected if the \"Resync a Data Services MongoDB Member\" procedure has recently been executed."
```